templates/repo/commit_page.tmpl

```diff
@@ -145,10 +145,18 @@
 		</div>
 		<div class="ui attached segment gt-df gt-ac gt-sb gt-py-2 commit-header-row gt-fw {{$class}}">
 				<div class="gt-df gt-ac author">
+					{{/* .Author: User model */}}
+					{{/* .Commit.Author: Author information stored in commit, see modules/git/Commit, modules/git/Signature */}}
+					{{/* .Verification.SigningUser: User model, see modules/git/CommitVerification */}}
 					{{if .Author}}
 						{{avatar $.Context .Author 28 "gt-mr-3"}}
-						{{if .Author.FullName}}
-							<a href="{{.Author.HomeLink}}"><strong>{{.Author.FullName}}</strong></a>
+						{{if .Author.Name}}
+							{{if eq .Commit.Author.Name .Author.Name}}
+								<a href="{{.Author.HomeLink}}"><strong>{{.Commit.Author.Name}}</strong></a>
+							{{else}}
+								{{/* If the account name and username don't match, use an alternative "also known as" format */}}
+								<a href="{{.Author.HomeLink}}"><strong>{{.Commit.Author.Name}} ({{.Author.Name}})</strong></a>
+							{{end}}
```

Comment:
(follow my previous comment and question)

Here the logic is changed: before `{{if .Author.FullName}}` , now `{{if .Author.Name}}`, while I think `.Author.Name` should always exist, it is the username, so this `if` is a no-op?


And, here it never shows user's full name, it it done on purpose?  (quote my previous question: when to use user.Name , when to use user.FullName)